src/Online_Bookstore/Dashboard/BookStore.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import ReviewedBooks from "../ReviewSection/ReviewedBooks";
+import BooksView from "./BooksView";
+import BookSearch from "./BookSearch";
+function BookStore() {
+  const [result, setResult] = useState([]);
+  const navigate = useNavigate();
+  const getBookData = async () => {
+    try {
+      const response = await fetch(`https://api.itbook.store/1.0/new`);
+      const bookData = await response.json();
+      setResult(bookData.books);
+    } catch (err) {
+      console.log(err);
+    }
+
+  };
+  useEffect(() => {
```

Comment:
Seems like this particular code snippet has been used above as well. Instead of writing the same code multiple times in different files, you can create a custom hook (if any react hook is involved) or create a utility file (if any business logic with the use of any react instance)